files/build_deploy.py

```diff
@@ -206,9 +213,9 @@ def notify_error(stage, error):
         notify_error('install', C.output)
 
     try:
-        syslog.syslog("Build of {}: bundle exec middleman build".format(name))
-        result = subprocess.check_output(['bundle', 'exec', 'middleman',
-                                          'build', '--verbose'])
+        command = builder_commands[config['builder']]
```

Comment:
In fact, i wonder if we need a config file, or if we can also try to be smart and autodetect. Like, if there is a gem file and it contains middleman, then we know what to do. I can be added later however, but I wanted to dump that idea so I do not forget.